topics/single-cell/tutorials/scrna-data-ingest/tutorial.md

```diff
@@ -0,0 +1,561 @@
+>
+> 3. **Rename**  {% icon galaxy-pencil %} the dataset `Barcodes tabular`.
+>    
+> 4. Repeat the same with `Transcripts` file and rename it as `Transcripts tabular`.
+> 
+{: .hands_on}
+
+Now let's select the number of the reads we would like to keep. It's totally up to you, we choose 100000 here.
+
+> <hands-on-title> Downsampling </hands-on-title>
+>
+> 1. {% tool [Select last](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_tail_tool/1.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Text file"*:  `Barcodes tabular` (output of **FASTQ to Tabular** {% icon tool %})
+>    - *"Operation"*: `Keep last lines`
+>    - *"Number of lines"*: `100000`
+>
+> 2. **Rename** {% icon galaxy-pencil %} the dataset `Barcodes cut`.
+>    
+> 3. Repeat the same with `Transcripts tabular` file and rename it as `Transcripts cut`
+> 
+{: .hands_on}
+
+All done, now we just need to go back to FASTQ from Tabular again!
+
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [Tabular to FASTQ](toolshed.g2.bx.psu.edu/repos/devteam/tabular_to_fastq/tabular_to_fastq/1.1.5) %} with the following parameters:
+>    - {% icon param-file %} *"Tabular file to convert"*: `Barcodes cut` (output of **Select last** {% icon tool %})
+>    - *"Identifier column"*: `c1`
+>    - *"Sequence column"*: `c2`
+>    - *"Quality column"*: `c3`
+>
+> 2. **Rename** {% icon galaxy-pencil %} the dataset `Downsampled barcode/UMI read`.
+>    
+> 3. Repeat the same with `Transcripts cut` file and rename it as `Downsampled transcript read`
+> 
+{: .hands_on}
+
+And that's all! Your downsampled data is ready to use. You can check your answers in this [example history](https://usegalaxy.eu/u/j.jakiela/h/how-to-downsample-fastq-files) or if you want to accelerate this process, feel free to use the [workflow](https://singlecell.usegalaxy.eu/u/j.jakiela/w/workflow-constructed-from-history-copy-of-cs1generating-a-single-cell-matrix-using-alevin---how-to-downsample) next time!
+
+
+# Format conversion
+
+In Galaxy Toolshed there is a wonderful tool called {% tool [SCEasy](toolshed.g2.bx.psu.edu/repos/iuc/sceasy_convert/sceasy_convert/0.0.7+galaxy1) %} which allows to convert between common single cell formats, such as:
+- AnnData to CellDataSet (CDS)
+- AnnData to Seurat
+- Loom to AnnData
+- Loom to SingleCellExperiment (SCE)
+- SingleCellExperiment (SCE) to AnnData
+- SingleCellExperiment (SCE) to Loom
+- Seurat to AnnData
+- Seurat to SingleCellExperiment (SCE)
+
+> <warning-title>Two SCEasy tools</warning-title>
+>  
+> The updated SCEasy tool is called [**SCEasy Converter**](toolshed.g2.bx.psu.edu/repos/iuc/sceasy_convert/sceasy_convert/0.0.7+galaxy1) and it's only available on usegalaxy.eu. The second tool is called [**SCEasy convert**](toolshed.g2.bx.psu.edu/repos/ebi-gxa/sceasy_convert/sceasy_convert/0.0.5+galaxy1) and it works on usegalaxy.org, however has limited conversion options.
+> 
+{: .warning}
```

Comment:
Whoops, we actually wanted to avoid this

The idea was that the new sceasy tool in the IUC repo would be a direct update of the EBI-GXA repo (indeed, that's what I based it on). I pushed the tool to IUC because I don't have ebi access and long-term wise, iuc made more sense.

Maybe we retire the old sceasy tool @pcm32 to avoid any confusion?

Reply:
Please PR that. Anyone can PR to that repo, I think it might have been better to do the change there and avoid the change in identifier for the tool. I would appreciate if we can avoid this pattern in the future, thanks!